docs/index.md

```diff
@@ -41,7 +41,6 @@ Sending a donation using our links below is **extremely helpful** in offsetting
 
 If you'd rather not donate (_which is okay!_), there are other ways you can help support us:
 
-- [Pi-hole Swag Store](https://pi-hole.net/shop/)
```

Comment:
Removed this since it just redirected to the main site

Reply:
Yes, this is gone IIRC. Pinging @dschaper

Reply:
Yeah, it's gone.